projects/extension/src/background/protocol.ts

```diff
@@ -52,7 +52,6 @@ export type ToContentScript = ToContentScriptWellKnownChain
 // Response to a {ToExtensionGetWellKnownChain}
 export interface ToContentScriptWellKnownChain {
   type: "get-well-known-chain"
-  chainName: string
```

Comment:
It was necessary in order to match the proper response with the proper request. Removed as no longer necessary.